mmv1/products/cloudbuild/api.yaml

```diff
@@ -28,6 +28,138 @@ apis_required:
+    description: |
+      BitbucketServerConfig represents the configuration for a Bitbucket Server.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'bitbucketServerConfigId'
+        required: true
+        url_param_only: true
+        input: true
+        description: |
+          The ID to use for the BitbucketServerConfig, which will become the final component of the BitbucketServerConfig's resource name.
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        url_param_only: true
+        input: true
+        required: true
+        description: |
+          The location of this bitbucket server config.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The resource name for the config.
+      - !ruby/object:Api::Type::String
+        name: 'hostUri'
+        required: true
+        description: |
+          Immutable. The URI of the Bitbucket Server host. Once this field has been set, it cannot be changed.
+          If you need to change it, please create another BitbucketServerConfig.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'secrets'
+        required: true
+        description: |
+          Secret Manager secrets needed by the config.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'adminAccessTokenVersionName'
+            required: true
+            description: |
+              The resource name for the admin access token's secret version.
+          - !ruby/object:Api::Type::String
+            name: 'readAccessTokenVersionName'
+            required: true
+            description: |
+              The resource name for the read access token's secret version.
+          - !ruby/object:Api::Type::String
+            name: 'webhookSecretVersionName'
+            required: true
+            input: true
+            description: |
+              Immutable. The resource name for the webhook secret's secret version. Once this field has been set, it cannot be changed.
+              If you need to change it, please create another BitbucketServerConfig.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        output: true
+        description: |
+          Time when the config was created.
+      - !ruby/object:Api::Type::String
+        name: 'username'
```

Comment:
Just want to confirm the behavior of this field. If we don't specify this field explicitly, will the API return a value for this field?

Reply:
No, it needs this field from the user, it will return an error if not specified (it is the bitbucket server user name).

Reply:
Sorry, I missed `required: true` setting somehow and thought it was an optional field, and that's why I asked the question. Thanks for confirming though!